src/components/exercise-identifier-link.cjsx

```diff
@@ -23,8 +23,5 @@ ExerciseIdentifierLink = React.createClass
       <span className='exercise-identifier-link'>
         ID# {@props.exerciseId} | <a target="_blank" href={url}>Report an error</a>
       </span>
-      <p>
-        { _.map(@props.related_content, @buildLabel) }
-      </p>
     </div>
```

Comment:
This seems like it's removing the `Comes from ...` message.  Is that intentional?